examples/PyTorch/Inference/hf_transformers/tests/test_wrapper_model.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from blade_adapter import WrapperModel, create_model, trace_model
+from transformers import GPT2LMHeadModel
+
+
+class WrapperModelTest(unittest.TestCase):
```

Comment:
Where is the script that triggers those tests?

Reply:
These tests haven't been added to CI yet. Considering some tests need to download models(typically hundreds of MBs per model) and run disc compilation, and code change in this adapter won't effect disc itself, we may want to setup a separated test schedule for this?